### PR TITLE
Remove dependency on nightly Rust

### PR DESCRIPTION
**Pull request by CohenArthur**
- **ffi-polonius: Remove usage of extern types.**
- **github: Remove nightly Rust installation**

@dkm sorry for not making this earlier!

### CODE REVIEW

**Comment by CohenArthur** [comment]
@braw-lee this affects you a little bit so pinging you :)

File: .github/workflows/ccpp.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
                   g++-multilib \
                   dejagnu;
           # install Rust directly using rustup
-          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain=nightly;
+          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain=1.72.0;
```
Comment (dkm) [inline .github/workflows/ccpp.yml:45]: Out of curiosity, any reason for using 1.72 here? Latest is 1.80, so that's several  release behind.

Comment (CohenArthur) [reply .github/workflows/ccpp.yml:45]: not most stable, most recent*

**Review by dkm — APPROVED** [review]

**Review by P-E-P — CHANGES_REQUESTED** [review]
We probably need to modify the bootstrap configuration too.

**Comment by dkm** [comment]
> We probably need to modify the bootstrap configuration too.

Oh good point, missed it.

**Comment by CohenArthur** [comment]
> We probably need to modify the bootstrap configuration too.

thanks! fixed in a8727ce

**Review by P-E-P — APPROVED** [review]

**Comment by CohenArthur** [comment]
Fixed the commit message

**Pull request merged**